tests/mod-resolver/issue-5198/lib/b.rs

```diff
@@ -0,0 +1 @@
+fn main(        ) {   println!("Hello World!")                         }
```

Comment:
Fully understand the utility of copy/pasta but for future reference I find it's helpful to make these types of files even subtly different (even just a different fn name) as it stands out more on the off chance we end up mapping the wrong files to modules.

Understand that wouldn't be caught by the particular test here given the nature of the test, just wanted to make a note